Update supportassistant_change_log.md (#3856)
SkylineCommunications/dataminer-docs: release-notes/DxMs/supportassistant_change_log.md

# Support Assistant change log

#### 30 October 2024 - Enhancement - SupportAssistant 1.7.1 - Include SLAutomation_BAK.txt in file offloads [ID_41281]

> From now on the SLAutomation_BAK.txt file is included in the file offloads, so DataMiner restarts are covered.

Reviewer: ```suggestion
From now on, the SLAutomation_BAK.txt file is also included in the file offloads.
```
